packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -159,7 +162,7 @@ export default function (
             sourceMap: !!(task.sourceMappings || task.sourceMap),
           });
 
-          _.extend(task.options, taskOpts);
+          extend(task.options, taskOpts);
```

Comment:
I don't know if it's `intended` for `taskOpts` prototype keys to be copied, so I'm doing the safe thing here and maintaining the exact same behavior. If this PR is merged, then I'll do another pass at removing some of these utils like in #5043.